activestorage/dummy_data.py

```diff
@@ -18,6 +18,13 @@ def _make_data(n=10):
 
     return dd, data
 
+def make_pathological_ncdata(filename='test_pathological_missing.nc', chunksize=(3,3,1), n=10):
+    """ 
+    Makes a test dataset based on the default vanilla dataset, but which includes
+    half missing values
+    """
+    return make_ncdata(filename, chunksize, n, compression=None, missing=-999., pathological=True)
```

Comment:
```suggestion
    return make_ncdata(filename, chunksize, n, compression=None, missing=-999., missing_data=True)
```

Reply:
Would prefer not `missing_data` since that's implicit with `missing`. Perhaps `extra_missing_data`?

Reply:
done "partially_missing"